libraries/botbuilder-ai/botbuilder/ai/luis/luis_recognizer.py

```diff
@@ -8,15 +8,21 @@
 from azure.cognitiveservices.language.luis.runtime.models import LuisResult
 from msrest.authentication import CognitiveServicesCredentials
 
-from botbuilder.core import BotAssert, IntentScore, RecognizerResult, TurnContext
+from botbuilder.core import (
+    BotAssert,
+    IntentScore,
+    Recognizer,
+    RecognizerResult,
+    TurnContext,
+)
 from botbuilder.schema import ActivityTypes
 
 from . import LuisApplication, LuisPredictionOptions, LuisTelemetryConstants
 from .activity_util import ActivityUtil
 from .luis_util import LuisUtil
 
 
-class LuisRecognizer:
+class LuisRecognizer(Recognizer):
```

Comment:
Hmm - not sure if this is for Luis V3 or old one, but looking at C#, there's another level of TelemetryRecognizer if you want to be faithful to that..

Reply:
Old one. Yes, but that interface only provide more method overloading (not supported in python) and attributes (not supported in the ABC pattern), making it redundant.

The idea here is to provide an interface that can be used at testing, where this ABC is sufficient.